packages/rspeedy/core/src/plugins/sourcemap.plugin.ts

```diff
@@ -99,6 +109,7 @@ function applySourceMapPlugin(
       noSources,
       namespace: output?.devtoolNamespace,
       publicPath,
+      debugIds,
```

Comment:
How about implementing it similarly to how it's done in Webpack (https://github.com/webpack/webpack/pull/18947)?

```js
const debugIds = devtool.includes('debugids')
```